src/index.liquid

```diff
@@ -48,14 +48,14 @@
       </div>
     </div>
   </section>
-  <section id="news-banner" class="hero is-warning">
+  <section id="news-banner" class="hero is-success">
     <div class="hero-body">
       <div class="container content has-text-centered is-size-5">
-        <a href="/news/#securing-shared-information">
-          <span>NEWS: security research in partnership with </span>
-          <img src="/media/logo_nlnet.svg" alt="NLnet" />
+        <a href="/news/#news_2022-08-30">
+          <span>NEWS: Federated Timesheets in partnership with </span>
+          <img src="./media/logo-pondersource.jpeg" alt="Ponder Source" />
           <span> and </span>
-          <img src="/media/NGIAssure.svg" alt="NGI Assure" />
+          <img src="./media/EvoluData-logo-web.png" alt="EvoluData" />
```

Comment:
I pushed some refinements of the images, hopefully you approve when the preview has been regenerated